File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxOnes = 0;
        int curOnes = 0;
        for (int num : nums) {
            if (num == 1)
                curOnes++;
            else {
                maxOnes = Math.max(curOnes, maxOnes);
                curOnes = 0;
            }
        }
        maxOnes = Math.max(curOnes, maxOnes);
        return maxOnes;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max1 = 0
        cur1 = 0
        for num in nums:
            if num == 0:
                max1 = max(max1, cur1)
                cur1 = 0
            else :
                cur1 +=1
        return max(max1, cur1)
        